src/widgets/copy-to-clipboard-button.js

```diff
@@ -65,19 +63,18 @@ export default class CopyToClipboardButton extends React.Component {
         </Localized>
       );
     }
-    return (
-      <Stack stretch selectedIndex={selectedIndex} className={className}>
```

Comment:
Oh yeah, and I meant to mention in my earlier comments that I found some of the CSS stuff a bit easier after dropping the Stack component (which was the main cause of the failed unit tests, so that reminded me)